File: Entities/Concrete/Question.cs
using System;
using System.Collections.Generic;
using Core.Entities;
using Entities.Concrete;
using Entities.Enums;


public class Question : IEntity
{

    public string Id { get; set; }
    public string QuestionContent { get; set; }
    public string QuestionImage { get; set; }
    public int QuestionDuration { get; set; }
    public int QuestionScore { get; set; }
    public QuestionOption QuestionOptions { get; set; }
    public QuestionType QuestionType { get; set; }
    public AnswerType CorrectAnswer { get; set; }
}

File: UIPreparation/UI/src/app/models/Question.ts
import { QuestionType } from "./QuestionType";

export class Question{
  id:number;
  questionContent:string;
  questionImage:string;
  questionDuration:number;
  questionScore:number;
  isTimeOver:boolean;
  quizId:number;
  questionType: QuestionType;
}
